apps/etterlatte-behandling/src/main/kotlin/behandling/GenerellBehandlingService.kt

```diff
@@ -126,6 +141,65 @@ class RealGenerellBehandlingService(
         return hentBehandling(behandlingId)?.toDetaljertBehandling()
     }
 
+    override suspend fun hentBehandlingMedEnkelPersonopplysning(
+        behandlingId: UUID,
+        saksbehandler: Saksbehandler,
+        accessToken: String,
+        opplysningstype: Opplysningstype
+    ): BehandlingMedGrunnlagsopplysninger<Person> {
+        val behandling = hentBehandling(behandlingId)?.toDetaljertBehandling()!!
+        val sakId = behandling.sak
+        return coroutineScope {
+            val personopplysning = async {
+                grunnlagKlient.finnPersonOpplysning(
+                    sakId,
+                    opplysningstype,
+                    accessToken
+                )
+            }
+
+            BehandlingMedGrunnlagsopplysninger(
+                id = behandling.id,
+                soeknadMottattDato = behandling.soeknadMottattDato,
+                personopplysning = personopplysning.await()
+            ).also {
+                personopplysning.await()?.fnr?.let { loggRequest(saksbehandler, it) }
```

Comment:
Hvorfor ny `.await`? Kan du ikke hente ut fra `it`?

Reply:
Det kan du.